<robot name="table">
  <contact name="top">
    <link name="base_link"/>
    <point>
      +0.7 0.6 0.68
      +0.7 -0.6 0.68
      -0.1 -0.6 0.68
      -0.1 0.6 0.68
    </point>
    <shape>
      4 3 2 1 0
    </shape>
  </contact>

  <contact name="side">
    <link name="base_link"/>
    <point>
      +0.7 0.6 0.68
      +0.7 0.6 0.00
      0.025 0.6 0.68
      0.025 0.6 0.00
    </point>
    <shape>
      4 3 2 1 0
    </shape>
  </contact>

  <contact name="front">
    <link name="base_link"/>
    <point>
      +0.8 0.6 0.68
      +0.8 0.6 0.00
      +0.8 -0.6 0.68
      +0.8 -0.6 0.00
    </point>
    <shape>
      4 3 2 1 0
    </shape>
  </contact>
  
  <handle name="contact_01" clearance="0.05">
    <position xyz="0.5 0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_02" clearance="0.05">
    <position xyz="0.5 0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_03" clearance="0.05">
    <position xyz="0.5 0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_04" clearance="0.05">
    <position xyz="0.5 -0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_05" clearance="0.05">
    <position xyz="0.5 -0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_06" clearance="0.05">
    <position xyz="0.5 -0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

    <handle name="contact_07" clearance="0.05">
    <position xyz="0.4 0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_08" clearance="0.05">
    <position xyz="0.4 0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_09" clearance="0.05">
    <position xyz="0.4 0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_10" clearance="0.05">
    <position xyz="0.4 -0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_11" clearance="0.05">
    <position xyz="0.4 -0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_12" clearance="0.05">
    <position xyz="0.4 -0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_13" clearance="0.05">
    <position xyz="0.3 0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_14" clearance="0.05">
    <position xyz="0.3 0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_15" clearance="0.05">
    <position xyz="0.3 0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_16" clearance="0.05">
    <position xyz="0.3 -0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_17" clearance="0.05">
    <position xyz="0.3 -0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_18" clearance="0.05">
    <position xyz="0.3 -0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_19" clearance="0.05">
    <position xyz="0.2 0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_20" clearance="0.05">
    <position xyz="0.2 0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_21" clearance="0.05">
    <position xyz="0.2 0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_22" clearance="0.05">
    <position xyz="0.2 -0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_23" clearance="0.05">
    <position xyz="0.2 -0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_24" clearance="0.05">
    <position xyz="0.2 -0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_25" clearance="0.05">
    <position xyz="0.1 0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_26" clearance="0.05">
    <position xyz="0.1 0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_27" clearance="0.05">
    <position xyz="0.1 0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_28" clearance="0.05">
    <position xyz="0.1 -0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_29" clearance="0.05">
    <position xyz="0.1 -0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_30" clearance="0.05">
    <position xyz="0.1 -0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_31" clearance="0.05">
    <position xyz="0.0 0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_32" clearance="0.05">
    <position xyz="0.0 0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_33" clearance="0.05">
    <position xyz="0.0 0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_34" clearance="0.05">
    <position xyz="0.0 -0.1 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_35" clearance="0.05">
    <position xyz="0.0 -0.3 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

  <handle name="contact_36" clearance="0.05">
    <position xyz="0.0 -0.5 0.67" xyzw="-0.5 0.5 0.5 0.5"/>
    <link name="base_link"/>
    <mask> 1 1 1 0 1 1 </mask>
  </handle>

</robot>
